index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <section>
+      <header>
+        <nav>
+          <a class="logo" href="">My Logo</a>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="Menu">
+          </div>
+      </nav>
+    </header>
+        <div class="topimg">
+            <div class="imgorder imgorder1">
+              <img class="backimg backimg1" src="./images/Ellipse 11.png" alt="">
+              <img class="backimg backimg2" src="./images/Rectangle 4.png" alt="">
+            </div>
+            <div class="imgorder imgorder2">
+              <img class="backimg backimg3" src="./images/Rectangle 7.png" alt="">
+              <img class="backimg backimg4" src="./images/Ellipse 5.png" alt="">
+            </div>
+            <img class="icon" src="./images/Ellipse 7.png" alt="">
+            <div class="imgorder imgorder3">
+              <img class="backimg backimg5" src="./images/Rectangle 4.png" alt="">
+              <img class="backimg backimg6" src="./images/Ellipse 3.png" alt="">
+            </div>
+        </div>
+        <h1>I’m kirubel Glad <br/> to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>LET’S CONNECT</p>
+        <ul class="social">
+          <li><a href="#"><img src="./images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="./images/" alt=""></a></li>
```

Comment:
Well done addressing the display issue as mentioned by the previous reviewer 👏🏽 but here you are using an empty image, it's either for spacing purposes or you just forgot to put the path there 😜 either way it won't be necessary like that. either change it so that it shows actual content or remove it. ❌